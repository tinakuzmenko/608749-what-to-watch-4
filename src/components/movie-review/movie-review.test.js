import React from 'react';
import renderer from 'react-test-renderer';
import MovieReview from './movie-review';

const review = {
  id: 4563456345,
  user: {
    id: 1223,
    name: `Amanda Greever`,
  },
  rating: 8.0,
  comment: `I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.`,
  date: `2019-05-08T14:13:56.569Z`,
};

describe(`MovieReview`, () => {
  it(`Should render correctly`, () => {
    const tree = renderer
      .create(<MovieReview
        movieReview={review}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
