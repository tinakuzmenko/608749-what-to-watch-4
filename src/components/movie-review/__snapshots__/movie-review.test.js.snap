// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieReview Should render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Amanda Greever
      </cite>
      <time
        className="review__date"
        dateTime="2019-05-08"
      >
        May 8, 2019
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    8,0
  </div>
</div>
`;
